pyspark/bigdl/models/inception/inception.py

```diff
@@ -0,0 +1,313 @@
+                           config[1][1],
+                           1, 1, 1, 1).set_init_method(weight_init_method=Xavier())
+            .set_name(name_prefix + "1x1"))
+    conv1.add(ReLU(True).set_name(name_prefix + "relu_1x1"))
+    concat.add(conv1)
+    conv3 = Sequential()
+    conv3.add(SpatialConvolution(input_size, config[2][1], 1, 1, 1, 1).set_init_method(weight_init_method=Xavier())
+              .set_name(name_prefix + "3x3_reduce"))
+    conv3.add(ReLU(True).set_name(name_prefix + "relu_3x3_reduce"))
+    conv3.add(SpatialConvolution(config[2][1], config[2][2],
+                                 3, 3, 1, 1, 1, 1).set_init_method(weight_init_method=Xavier())
+              .set_name(name_prefix + "3x3"))
+    conv3.add(ReLU(True).set_name(name_prefix + "relu_3x3"))
+    concat.add(conv3)
+    conv5 = Sequential()
+    conv5.add(SpatialConvolution(input_size,
+                                 config[3][1], 1, 1, 1, 1).set_init_method(weight_init_method=Xavier())
+              .set_name(name_prefix + "5x5_reduce"))
+    conv5.add(ReLU(True).set_name(name_prefix + "relu_5x5_reduce"))
+    conv5.add(SpatialConvolution(config[3][1],
+                                 config[3][2], 5, 5, 1, 1, 2, 2).set_init_method(weight_init_method=Xavier())
+              .set_name(name_prefix + "5x5"))
+    conv5.add(ReLU(True).set_name(name_prefix + "relu_5x5"))
+    concat.add(conv5)
+    pool = Sequential()
+    pool.add(SpatialMaxPooling(3, 3, 1, 1, 1, 1,
+                               to_ceil=True).set_name(name_prefix + "pool"))
+    pool.add(SpatialConvolution(input_size, config[4][1], 1, 1, 1, 1).set_init_method(weight_init_method=Xavier())
+             .set_name(name_prefix + "pool_proj"))
+    pool.add(ReLU(True).set_name(name_prefix + "relu_pool_proj"))
+    concat.add(pool).set_name(name_prefix + "output")
+    return concat
+
+
+def Inception_v1_NoAuxClassifier(class_num):
+    model = Sequential()
+    model.add(SpatialConvolution(3, 64, 7, 7, 2, 2, 3, 3, 1, False)
+              .set_init_method(weight_init_method=Xavier(), bias_init_method=Zeros())
+              .set_name("conv1/7x7_s2"))
+    model.add(ReLU(True).set_name("conv1/relu_7x7"))
+    model.add(SpatialMaxPooling(3, 3, 2, 2, to_ceil=True).set_name("pool1/3x3_s2"))
+    model.add(SpatialCrossMapLRN(5, 0.0001, 0.75).set_name("pool1/norm1"))
+    model.add(SpatialConvolution(64, 64, 1, 1, 1, 1).set_init_method(weight_init_method=Xavier())
+              .set_name("conv2/3x3_reduce"))
+    model.add(ReLU(True).set_name("conv2/relu_3x3_reduce"))
+    model.add(SpatialConvolution(64, 192, 3, 3, 1, 1, 1, 1).set_init_method(weight_init_method=Xavier())
+              .set_name("conv2/3x3"))
+    model.add(ReLU(True).set_name("conv2/relu_3x3"))
+    model.add(SpatialCrossMapLRN(5, 0.0001, 0.75).set_name("conv2/norm2"))
+    model.add(SpatialMaxPooling(3, 3, 2, 2, to_ceil=True).set_name("pool2/3x3_s2"))
+    model.add(Inception_Layer_v1(192, scala_T([scala_T([64]), scala_T(
```

Comment:
why we need scala_T? it looks complicated

Reply:
it mimic table in scala. In python, we could use python list directly, but the index comes from 0. To be consistent with scala model, create this structure. And I changed the name from scala_T to T.